lib/noticed/delivery_methods/fcm.rb

```diff
@@ -23,6 +23,15 @@ def send_notification(device_token)
         end
       end
 
+      def format_notification(device_token)
```

Comment:
This is a good idea. 👍

Reply:
Thank you! It's your idea originally :grinning:

Reply:
LOL